content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -434,7 +434,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   to discover plugins such as [CSI volume drivers](/docs/concepts/storage/volumes/#csi).
 - `KubeletPodResources`: Enable the kubelet's pod resources grpc endpoint.
    See [Support Device Monitoring](https://git.k8s.io/community/keps/sig-node/compute-device-assignment.md) for more details.
-- `LegacyNodeRoleBehavior`: When disabled, legacy behavior in service load balancers and node disruption will ignore the `node-role.kubernetes.io/master` label in favor of the feature-specific labels.
+- `LegacyNodeRoleBehavior`: When disabled, legacy behavior in service load balancers and node disruption will ignore the `node-role.kubernetes.io/master` label in favor of the feature-specific labels. This feature gate will be set to disabled in the 1.20 release, and the labels provided under the NodeDisruptionExclusion and ServiceNodeExclusion feature gates will be used to provide the equivalent behavior.
```

Comment:
Does this imply that `ServiceNodeExclusion` and `NodeDisruptionExclusion` are confirmed for GA in v1.20?

Reply:
The general guide line is avoiding predictions about future.

Reply:
Sure, the release notes will contain that awareness and that's the KEP plan.  Since this is an important change for deployers to make, we might want a separate document on it as part of either release notes or the documentation (migrating away from the previous legacy behavior).

Reply:
Essentially: don't commit to this going GA in 1.20 unless that's a confirmed thing.

Regardless of which release switches off the legacy behavior, adding a Task page on how to migrate away would definitely be a nice-to-have.